.codesandbox/sandbox/src/App.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import {
+  Session as AuthSession,
+  getClientAuthenticationWithDependencies
+} from "@inrupt/solid-client-authn-browser";
+
+// const REDIRECT_URL = "https://podpecker.zwifi.eu"
+// const REDIRECT_URL = "http://localhost:3000";
+const REDIRECT_URL = "https://ycos2.csb.app/";
```

Comment:
I don't think this will work, given that a new sandbox is created for every PR. Possibly just `window.location`?

Reply:
Good idea, `window.location` does the trick. Implemented in 812e598c79760c20176484d92b3ee1eb219c788f.